docs/_ug/About.md

```diff
@@ -1,6 +1,15 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-FoodRem is an Inventory Management System that empowers small food and beverage (F&B) restaurant managers to manage inventory and obtain insights from inventory data. As a restaurant manager, leverage upon FoodRem's Inventory Management System, where you can manage your inventory during your daily operations. Utilize FoodRem's flexible tagging system to help you organize your inventory according to your business needs. Finally, streamline your business decisions by deriving insights from your inventory usage through FoodRem's statistics, keeping track of vital data such as food wastage.
+FoodRem is an Inventory Management System that empowers small food and beverage (F&B) restaurant managers to manage 
+inventory and obtain insights from inventory data. As a restaurant manager, leverage upon FoodRem's Inventory Management 
```

Comment:
Is there a reason why we are splitting up these into seperate lines?

Reply:
Easier to see when coding. It does not affect how the markdown file is displayed

Reply:
It will affect depending on which renderer is used. Also, this violates our [coding standards for markdown](https://se-education.org/guides/conventions/markdown.html)

Reply:
Ah, I did not know that, will revert